odoo/tools/test_reports.py

```diff
@@ -121,7 +121,7 @@ def log_test(msg, *args):
         act_model, act_id = action._name, action.id
     else:
         assert isinstance(action_id, int)
-        act_model = 'ir.action.act_window'     # assume that
+        act_model = 'ir.actions.act_window'     # assume that
         act_id = action_id
         act_xmlid = '<%s>' % act_id
```

Comment:
It's only used in one place, maybe remove the `our_module` and `action_id: int` stuff entirely?

Reply:
maybe a bit out of scope, this one was already found by grep but looks like dead code anyway